<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomatedRssDownloader</class>
 <widget class="QDialog" name="AutomatedRssDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSS Downloader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QCheckBox" name="checkEnableDownloader">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enable Automated RSS Downloader</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="hsplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Download Rules</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeRuleBtn">
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addRuleBtn">
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listRules">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="ruleDefBox">
         <property name="title">
          <string>Rule Definition</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="checkRegex">
            <property name="text">
             <string>Use Regular Expressions</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Must Contain:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Must Not Contain:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblEFilter">
              <property name="text">
               <string>Episode Filter:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="lblEFilterStat">
              <property name="maximumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="lbl_mustnot_stat">
              <property name="maximumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineContains"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEFilter"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="lbl_must_stat">
              <property name="maximumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineNotContains"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Assign Label:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboLabel">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="saveDiffDir_check">
            <property name="text">
             <string>Save to a Different Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Save to:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineSavePath">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="browseSP">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string notr="true">...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="lblIgnoreDays">
              <property name="text">
               <string comment="... X days">Ignore Subsequent Matches for (0 to Disable)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinIgnorePeriod">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="suffix">
               <string> days</string>
              </property>
              <property name="maximum">
               <number>365</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lblLastMatch">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="lblAddPaused">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Add Paused:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboAddPaused">
              <item>
               <property name="text">
                <string>Use global settings</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Always</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Never</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Apply Rule to Feeds:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listFeeds"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Matching RSS Articles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeMatchingArticles">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="importBtn">
       <property name="text">
        <string>&amp;Import...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="text">
        <string>&amp;Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AutomatedRssDownloader</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AutomatedRssDownloader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveDiffDir_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveDiffDir_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineSavePath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveDiffDir_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseSP</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
